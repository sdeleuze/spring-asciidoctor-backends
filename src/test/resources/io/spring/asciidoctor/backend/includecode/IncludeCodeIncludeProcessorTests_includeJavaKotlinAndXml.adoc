= Test Document
:docs-java: {docdir}/javacode
:docs-kotlin: {docdir}/kotlincode
:docs-xml: {docdir}/xmlcode

[[some.convention-based-anchor]]
== Some Convention Based Anchor

include::code:SomeCode[]
