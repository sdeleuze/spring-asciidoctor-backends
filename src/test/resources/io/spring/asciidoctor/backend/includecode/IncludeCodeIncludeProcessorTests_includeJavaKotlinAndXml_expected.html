<div class="listingblock primary">
 <div class="title">
  Java
 </div>
 <div class="content">
  <pre class="highlight"><code class="language-java" data-lang="java">somejava
</code></pre>
 </div>
</div>
<div class="listingblock secondary">
 <div class="title">
  Kotlin
 </div>
 <div class="content">
  <pre class="highlight"><code class="language-kotlin" data-lang="kotlin">somekotlin
</code></pre>
 </div>
</div>
<div class="listingblock secondary">
 <div class="title">
  XML
 </div>
 <div class="content">
  <pre class="highlight"><code class="language-xml" data-lang="xml">somexml</code></pre>
 </div>
</div>